packages/build-tools/src/wasiDepsParser.ts

```diff
@@ -1,6 +1,24 @@
 import fs from 'fs';
 import { TargetWorld } from '../lib/wit_tools.js';
 import path from 'path';
+import { resolve } from 'node:path';
+import { platform } from 'node:process';
+const isWindows = platform === 'win32';
+
```

Comment:
I don't think this is technically copyrightable, but we should still give attribution. Luckily the license is the same, so a simple link is sufficient.
```suggestion

// Copied from https://github.com/bytecodealliance/ComponentizeJS/blob/e5bb0da0c1ce1ee87eed45c970121ed5fe8cca3a/src/componentize.js#L26-L39
```